packages/react-native/ReactCommon/jsc/JSCRuntime.cpp

```diff
@@ -452,7 +465,11 @@ std::string JSCRuntime::description() {
 }
 
 bool JSCRuntime::isInspectable() {
+#ifdef _JSC_HAS_INSPECTABLE
+  return JSGlobalContextIsInspectable(ctx_);
```

Comment:
Where are we using this `isInspectable` in a way that would be useful with JSC? I suspect it would actually be a regression as it might cause some of the CDP infra / UI to appear, without it actually working.

Reply:
That's a good point. I implemented it because we could. But if this means "conforms to CDP", then I'll undo this

Reply:
Undone, this goes back to returning false. I suppose if going through Safari Web Inspector, this isn't needed at all.